router/batchrouter/handle.go

```diff
@@ -576,6 +576,7 @@ func (brt *Handle) updateJobStatus(batchJobs *BatchedJobs, isWarehouse bool, err
 	transformedAtMap := make(map[string]string)
 	statusDetailsMap := make(map[string]*types.StatusDetail)
 	jobStateCounts := make(map[string]int)
+	jobIdConnectionDetailsMap := make(map[int64]*jobsdb.ConnectionDetailsT)
```

Comment:
What is the reason for using a pointer to a struct? Fro small struct is more efficient to pass them as is.

also (nitpik) `jobID` instead of `jobId` https://google.github.io/styleguide/go/decisions.html#initialisms

Reply:
I understand the reason we use the maps, but is a hack that we need to address some day

Reply:
I will change it for sourceId and destinationId as well